CMakeLists.txt

```diff
@@ -10,6 +10,8 @@ include_directories(bomberman sources)
 add_subdirectory(${PROJECT_SOURCE_DIR}/lib/wal)
 add_subdirectory(${PROJECT_SOURCE_DIR}/lib/Ray)
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O2")
```

Comment:
Is this really needed ?

Reply:
this should be add at the end when the game is already optimised (might add some more FPS)